packages/contracts/contracts/L1/ValidatorPool.sol

```diff
@@ -249,25 +256,40 @@ contract ValidatorPool is ReentrancyGuardUpgradeable, Semver {
      * @return Whether the bond has been successfully unbonded.
      */
     function _tryUnbond() private returns (bool) {
-        uint256 outputIndex = nextUnbondOutputIndex;
+        uint256 outputIndex;
+        uint128 bondAmount;
+        Types.Bond storage bond;
+        Types.CheckpointOutput memory output;
+
+        uint256 i = 0;
+        for (; i < MAX_UNBOND; ) {
+            outputIndex = nextUnbondOutputIndex + i;
+            bond = bonds[outputIndex];
+            bondAmount = bond.amount;
 
-        Types.Bond storage bond = bonds[outputIndex];
-        uint128 bondAmount = bond.amount;
-        if (block.timestamp >= bond.expiresAt && bondAmount > 0) {
-            delete bonds[outputIndex];
+            if (block.timestamp >= bond.expiresAt && bondAmount > 0) {
```

Comment:
There are no cases where the amount value is 0 on the bond map. 
```suggestion
            if (block.timestamp >= bond.expiresAt) {
```

Reply:
I think this logic is needed to check bond existence. If the bond of specific `outputIndex` does not exist, `bond.expiresAt` and `bond.amount` is initialized to 0.